spec/assignable_values/active_record_spec.rb

```diff
@@ -370,6 +370,211 @@ def save_without_validation(record)
 
     end
 
+    context 'when validating scalar attributes from a store_accessor' do
+
+      context 'without options' do
+
+        before :each do
+          @klass = Song.disposable_copy do
+            store :metadata, accessors: [:format], coder: JSON
+
+            assignable_values_for :format do
+              %w[mp3 wav]
+            end
+          end
+        end
+
+        it 'should validate that the attribute is allowed' do
+          @klass.new(:format => 'mp3').should be_valid
+          @klass.new(:format => 'disallowed value').should_not be_valid
+        end
+
+        it 'should use the same error message as validates_inclusion_of' do
+          record = @klass.new(:format => 'disallowed value')
+          record.valid?
+          errors = record.errors[:format]
+          error = errors.respond_to?(:first) ? errors.first : errors # the return value sometimes was a string, sometimes an Array in Rails
+          error.should == I18n.t('errors.messages.inclusion')
+          error.should == 'is not included in the list'
```

Comment:
What's the difference?

Reply:
C&P thing. Adapted the original line as well